.github/workflows/book.yml

```diff
@@ -0,0 +1,32 @@
+name: miden book
+
+on:
+  push:
+    branches:
+      - master
```

Comment:
missed this on the first review: we don't have a 'main` branch instead of `master`. Should it be fixed here?